model/PFAHandler.php

```diff
@@ -526,21 +526,21 @@ protected function setmore(array $values) {
     }
 
     /**
-     * store $this->values in the database
+     * save $this->values to the database
      *
      * converts values based on $this->struct[*][type] (boolean, password encryption)
      *
      * calls $this->storemore() where additional things can be done
      * @return bool - true if all values were stored in the database, otherwise false
      *     error messages (if any) are stored in $this->errormsg
      */
-    public function store() {
+    public function save() : bool {
```

Comment:
What's the reason for renaming `store()`, `beforestore()` and `storemore()`?

The problem I see is that external users of PostfixAdmin will have to adjust their code / additional classes, without giving any benefit. (In case you wonder - I know at least one case of PostfixAdmin usage with additional classes that won't fit for upstream because they are totally unrelated to mail handling.)

If you really want to rename these functions, please keep compatibility calls to their old names to avoid breaking those additional classes.

Reply:
preSave(), save() and postSave() seemed more descriptive and follow a more widespread naming convention.

I'll add redirection functions in to avoid issues.